src/runtime/aarch64.rs

```diff
@@ -32,11 +32,11 @@ pub enum __Feature {
 }
 
 pub fn detect_features<T: linux::FeatureQuery>(mut x: T) -> usize {
-    let value: usize = 0;
+    let mut value: usize = 0;
     {
         let mut enable_feature = |f| {
             if x.has_feature(&f) {
-                bit::set(value, f as u32);
+                value = bit::set(value, f as u32);
```

Comment:
thanks for fixing this! The x86 run-time did not have this issue, I wish there was an easy way to unify this code. https://github.com/rust-lang-nursery/stdsimd/blob/master/coresimd/src/runtime/x86.rs#L329